raiden-ts/tests/unit/raiden.spec.ts

```diff
@@ -654,4 +655,35 @@ describe('Raiden', () => {
       }),
     ).resolves.toEqual(channelOpenHash);
   });
+
+  test('getBalance', async () => {
```

Comment:
This and `getTokenBalance` were already added by https://github.com/raiden-network/light-client/pull/2697, so can be removed.

This also needs a rebase.

Reply:
Ok I will remove it.